src/Fixer/Alias/NoAliasFunctionsFixer.php

```diff
@@ -103,24 +176,57 @@ public function configure(array $configuration = null)
         foreach ($this->configuration['sets'] as $set) {
             if ('@all' === $set) {
                 $this->aliases = self::$internalSet;
+                $this->aliases = array_merge($this->aliases, self::$exifSet);
+                $this->aliases = array_merge($this->aliases, self::$ftpSet);
                 $this->aliases = array_merge($this->aliases, self::$imapSet);
+                $this->aliases = array_merge($this->aliases, self::$ldapSet);
                 $this->aliases = array_merge($this->aliases, self::$mbregSet);
+                $this->aliases = array_merge($this->aliases, self::$mysqliSet);
+                $this->aliases = array_merge($this->aliases, self::$ociSet);
+                $this->aliases = array_merge($this->aliases, self::$odbcSet);
+                $this->aliases = array_merge($this->aliases, self::$opensslSet);
+                $this->aliases = array_merge($this->aliases, self::$pcntlSet);
+                $this->aliases = array_merge($this->aliases, self::$pgSet);
+                $this->aliases = array_merge($this->aliases, self::$posixSet);
+                $this->aliases = array_merge($this->aliases, self::$snmpSet);
+                $this->aliases = array_merge($this->aliases, self::$sodiumSet);
                 $this->aliases = array_merge($this->aliases, self::$timeSet);
-                $this->aliases = array_merge($this->aliases, self::$exifSet);
 
                 break;
             }
 
             if ('@internal' === $set) {
                 $this->aliases = array_merge($this->aliases, self::$internalSet);
+            } elseif ('@exif' === $set) {
```

Comment:
Even before this PR the `if`/`elseif` block wasn't looking pretty - can we have an array `@setname` => self::$setValues` and iterate over it?

Reply:
Done!